lib/Clownface.js

```diff
@@ -58,7 +58,7 @@ class Clownface {
   }
 
   list () {
-    if (!this.term) {
+    if (!this.term && this.terms.length > 0) {
```

Comment:
Looks like it should be `||`

Reply:
`&&` was correct, but I simplified it a little bit. `!this.term` actually checks if `this.terms.length === 1`, so for `0` or `1` it was OK and for everything else the error gets thrown.